projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -55,11 +131,22 @@ type jobRequestResult = {
 export async function jobRunRequest(): Promise<jobRequestResult> {
   if (await checkIfVdkOptionDataIsDefined(VdkOption.PATH)) {
     try {
-      const data = await requestAPI<serverVdkOperationResult>('run', {
-        body: JSON.stringify(getJobDataJsonObject()),
-        method: 'POST'
-      });
-      return { message: data['message'], isSuccessful: data['message'] == '0' };
+      const initialResponse = await requestAPI<serverVdkOperationResult>(
+        'run',
+        {
+          body: JSON.stringify(getJobDataJsonObject()),
+          method: 'POST'
+        }
+      );
+      if (initialResponse.error) {
+        showError(initialResponse.error);
+        return { message: initialResponse.message, isSuccessful: false };
+      }
+      const finalResult = await pollForTaskCompletion('RUN');
```

Comment:
What about if two ppl run tasks in a notebook at the same time? 
does the task id not need to be unique to that run? 
the same would be true for the convert notebook. 
what happens if two ppl are trying to convert a notebook at the same time?

Reply:
We don't support running multiple tasks simultaneously at this point. When we decide to, we can modify the task runner appropriately and introduce UID or some unique prefix/suffix to the task id.

Reply:
but this will run on the jupyter server. so does one user block it for all other users? 
That seems like awful functionality.

Reply:
But different users will have the same task id on the server ?

Reply:
Done. Made use of UUID to do so.